src/scss/_type.scss

```diff
@@ -44,196 +44,294 @@
 }
 
 @mixin hText(
-  $size: $heading-size1,
+  $size: var(--font-size-xl),
   $color: $pure-black,
   $transform-style: capitalize,
-  $line-height: $heading-line-height-size1
+  $text-decoration: unset,
+  $font-style: unset,
+  $line-height: calc($size + var(--font-size-step))
 ) {
   @include DistinctDisplay;
 
   color: $color;
   font-size: $size;
+  font-style: $font-style;
   font-variation-settings: 'wght' 400;
```

Comment:
We should probably convert these to `font-weight` since our heading text is using Distinct which is not a variable font.